# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/crashtests.list
# Commit: 6d98d185a076
# Full Hash: 6d98d185a07684b3538dd0052cb14b67cba38568
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2021-11-05 09:34:21
# Regressor Bug: 1730695
# File Overlap Count: 1
# Description:
#   Bug 1730695 - Fix panic when casting large value to i32 rect r=gfx-reviewers,bradwerth,nical,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D128122
# ==============================================================================

diff -r bba94c79f3e1 -r 6d98d185a076 gfx/tests/crashtests/crashtests.list
--- a/gfx/tests/crashtests/crashtests.list	Thu Nov 04 19:20:21 2021 +0000
+++ b/gfx/tests/crashtests/crashtests.list	Thu Nov 04 19:44:32 2021 +0000
@@ -207,3 +207,4 @@
 load 1700232.html
 load 1704321-1.html
 load 1702638.html
+load 1730695.html